akka-docs/rst/scala/dispatchers.rst

```diff
@@ -153,6 +130,10 @@ And then using it:
 
 .. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#defining-fixed-pool-size-dispatcher
 
+Another example that uses the thread pool based on the number of cores (e.g. for CPU bound tasks)
+
+.. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#my-thread-pool-dispatcher-config
```

Comment:
Hmmm, but that's still the complex example though?
https://github.com/akka/akka/blob/master/akka-docs/rst/scala/code/docs/dispatcher/DispatcherDocSpec.scala#L86